crates/aptos/src/common/types.rs

```diff
@@ -1060,15 +1060,11 @@ pub struct MovePackageDir {
     pub bytecode_version: Option<u32>,
 
     /// Specify the version of the compiler.
-    ///
-    /// Currently hidden until the official launch of Compiler V2
-    #[clap(long, hide = true, value_parser = clap::value_parser!(CompilerVersion))]
+    #[clap(long, value_parser = clap::value_parser!(CompilerVersion))]
     pub compiler_version: Option<CompilerVersion>,
```

Comment:
Can we add to the doc comment:

Currently, default to V1 (or whatever it defaults to)